src/main/kotlin/graphics/scenery/volumes/EmptyNode.kt

```diff
@@ -0,0 +1,25 @@
+package graphics.scenery.volumes
+
+import graphics.scenery.DefaultNode
+import graphics.scenery.Light
+import graphics.scenery.net.Networkable
+
+class EmptyNode : DefaultNode("EmptyNode"),Networkable {
```

Comment:
Is the purpose of the `EmptyNode` simply to transmit the change from VDI generation to volume rendering? There should be a better way to do that. If scene contents need to be synchronized more generally, maybe a node can be created for all required information but doing it for a single data is not ideal

Reply:
I agree with @aryaman-gupta 👍